Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Protocol.json

```diff
@@ -13,13 +13,13 @@
     "locked": false,
     "name": "ASM - Protocol",
     "neverSetAsRequired": false,
-    "openEnded": false,
+    "openEnded": true,
```

Comment:
Please elaborate more about the changes:
What exactly was the issue?
How did it fix it?
did you make sure it solved the issue?

Reply:
explained in https://panw-global.slack.com/archives/C0489NUGBRB/p1706181195830429?thread_ts=1706173202.412929&cid=C0489NUGBRB